common/azure/src/main/kotlin/com/cosmotech/api/azure/adx/AzureDataExplorerClient.kt

```diff
@@ -82,6 +83,29 @@ class AzureDataExplorerClient(private val csmPlatformProperties: CsmPlatformProp
         }
     return healthBuilder.withDetail("baseUri", baseUri).build()
   }
+
+  public fun deleteDataFromScenarioRunId(
+      organizationId: String,
+      workspaceKey: String,
+      scenarioRunId: String
+  ) {
+    var databaseName = getDatabaseName(organizationId, workspaceKey)
```

Comment:
This variable can be declared as `val`. It is not modified anywhere else.
```suggestion
    val databaseName = getDatabaseName(organizationId, workspaceKey)
```